ledger/eval/eval.go

```diff
@@ -1647,7 +1701,7 @@ func (eval *BlockEvaluator) generateKnockOfflineAccountsList() {
 
 		if acctData.Status == basics.Online {
 			lastSeen := max(acctData.LastProposed, acctData.LastHeartbeat)
-			if isAbsent(eval.state.prevTotals.Online.Money, acctData.MicroAlgos, lastSeen, current) ||
+			if isAbsent(eval.state.prevTotals.Online.Money, acctData.MicroAlgosWithRewards, lastSeen, current) ||
```

Comment:
I don't recall thinking it through that way, but it seems true.

Reply:
We should do a little further checking. I *think* any account considered touched here will also be updated.  I think for weird cases like an axfer from A to B, only A is touched (since its algo balance changes when paying fee).  Similar stuff can happen with accounts in app calls. Probably/hopefully  `eval.state.modifiedAccounts()` are exactly the ones that have truly been touched and will therefore have updated balances.